packages/libllvm/build.sh

```diff
@@ -3,8 +3,8 @@ TERMUX_PKG_DESCRIPTION="Modular compiler and toolchain technologies library"
 TERMUX_PKG_LICENSE="NCSA"
```

Comment:
Seems that LLVM has switched its license to `Apache 2.0 with LLVM Exception` (https://releases.llvm.org/14.0.0/LICENSE.TXT). Should we change it here?

Reply:
Yep, I will change it.